### ISSUE TITLE
Log migration support

### ISSUE THREAD

**Issue by krasserm** [RBMHTechnology/eventuate-tools#5]
_From @krasserm on September 14, 2015 13:4_

_Copied from original issue: RBMHTechnology/eventuate#111_

**Comment by krasserm** [comment]
Depends on #100

**Comment by krasserm** [comment]
_From @hepin1989 on October 28, 2015 6:39_

is this online migrating?and will it be webui based?

**Comment by krasserm** [comment]
@hepin1989 what do you mean by _online migrating_?

**Comment by krasserm** [comment]
_From @hepin1989 on October 28, 2015 7:18_

Offline migrating means you need a maintenance window to do the migrate ,like schema rewrite.
online migrate means you don't need to stop the server,and it migrating while your old schema is there,but finally.but finally it will keep in sync ,and you could do the switch.

I have not look very into eventuate,but I think cause it's AP,it maybe come up as a service,which maybe have a web ui for the server side,and on the user side,it's just as it is.

In short,I think eventuate as a service more than lib.

**Comment by krasserm** [comment]
@hepin1989 I think #107 should answers your question(s).

**Comment by krasserm** [comment]
_From @hepin1989 on October 28, 2015 7:24_

@krasserm Thank you.
